### ISSUE TITLE
{Global} [RARE] Spawn after death issue

### ISSUE THREAD

**Issue by LucKYsHoTSzZz** [FadeCloudNW/issue-tracker#77]
There is a chance upon death that you instantly get tped back to the location you died in instead of spawn.

Considered: Low Priority due to the low chance.

[Post](https://fadecloudmc.com/forums/threads/stinkinqs-global-bug-report.55651/)

**Comment by Inkzzz** [comment]
Waiting on response asked on the forum post.
